### PR TITLE
Update 20.treatments.md

### PR DESCRIPTION
**Pull request by byrdjb**
### Description of the proposed additions or changes
Added the possibility of gastrointestinal symptoms, with or without pulmonary symptoms.

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.treatments.md [hunk 1]
```diff
@@ -12,8 +12,9 @@ In the following sections, we critically appraise the literature surrounding the
 ### Treatment of Symptoms
 
 The clinical picture of SARS-CoV-2 infection differs dramatically between individuals.
-Some are asymptomatic.
-Others experience mild COVID-19 symptoms, such as cough, sore throat and fever, while the most severe cases of COVID-19 include severe complications including pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death [@url:https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html].
+Some are asymptomatic, and many experience mild COVID-19 symptoms.
+Mild symptoms can include cough, sore throat and fever, or in some cases loss of appetite and vomiting [@doi:10.14309/ajg.0000000000000664], or a combination of respiratory and gastrointestinal symptoms.
```
Comment (rando2) [inline content/20.treatments.md:16]:
```suggestion
Mild symptoms commonly include fever and respiratory symptoms such as cough and sore throat, and, less commonly, gastrointestinal symptoms such as loss of appetite and vomiting [@doi:10.14309/ajg.0000000000000664]; some patients experience a combination of respiratory and gastrointestinal symptoms.
```
Just a suggestion, but perhaps there is a way to organize these symptoms more explicitly into categories

File: content/20.treatments.md [hunk 2]
```diff
@@ -12,8 +12,9 @@ In the following sections, we critically appraise the literature surrounding the
...
+The most severe cases of COVID-19 include severe complications including pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death [@url:https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html].
```
Comment (rando2) [inline content/20.treatments.md:17]:
```suggestion
The most severe cases of COVID-19 include severe complications such as pneumonia and Acute Respiratory Distress Syndrome (ARDS), which can lead to respiratory failure and death [@url:https://www.cdc.gov/coronavirus/2019-ncov/hcp/clinical-guidance-management-patients.html].
```

File: content/20.treatments.md (hunk 1 above)
Comment (byrdjb) [reply content/20.treatments.md]: Much better--thank you!

**Comment by rando2** [comment]
Note for self: PR 292 is still in text

**Pull request merged**